### PR TITLE
Adds changelog parameter and fancy version parameters into all-citus

### PR DESCRIPTION
**Pull request by gurkanindibay**
Added a new changelog parameter to manually add changelogs

Depends on https://github.com/citusdata/tools/pull/183

### CODE REVIEW

File: .github/workflows/update_package_properties.yml [hunk 1]
```diff
@@ -27,10 +30,10 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Install dependencies
-        run: sudo apt install libcurl4-openssl-dev libssl-dev
+        run: sudo apt-get update && sudo apt install libcurl4-openssl-dev libssl-dev python3-testresources
 
       - name: Clone tools branch
-        run: git clone -b v0.8.1 --depth=1  https://github.com/citusdata/tools.git tools
+        run: git clone -b update_properties_fixes --depth=1  https://github.com/citusdata/tools.git tools
```
Comment (hanefi) [inline .github/workflows/update_package_properties.yml]: needs to be updated

Comment (hanefi) [reply .github/workflows/update_package_properties.yml]: @gurkanindibay Please update the PR description to indicate that this PR is blocked on a tools PR.

If you do not provide a link, it requires digging everytime. I do not want to spend time going through tools repo PRs and see which one actually has `update_properties_fixes` as the base branch.

File: .github/workflows/update_package_properties.yml [hunk 2]
```diff
@@ -17,6 +17,9 @@ on:
       name:
         description: "Name to be written on changelogs"
         default: "Gurkan Indibay"
+      changelog_entry:
+        description: "If filled, changelog other than default entry will be written"
+        default: ""
```
Comment (hanefi) [inline .github/workflows/update_package_properties.yml:26]: You are missing `required` properties here. All inputs entries should have one.

File: .github/workflows/update_package_properties.yml (hunk 1 above)
Comment (gurkanindibay) [reply .github/workflows/update_package_properties.yml]: I've already added one in description https://github.com/citusdata/packaging/pull/782#issue-737431653

File: .github/workflows/update_package_properties.yml (hunk 2 above)
Comment (gurkanindibay) [reply .github/workflows/update_package_properties.yml:26]: Fixed

File: .github/workflows/update_package_properties.yml (hunk 1 above)
Comment (hanefi) [reply .github/workflows/update_package_properties.yml]: > I've already added one in description [#782 (comment)](https://github.com/citusdata/packaging/pull/782#issue-737431653)

No, I did that. Please provide links in the PR description in the future.

**Review by hanefi — DISMISSED** [review]
Don't forget to update the branch name after citusdata/tools#183 is merged

**Review by hanefi — APPROVED** [review]

**Pull request merged**